python_modules/dagster/dagster/_daemon/backfill.py

```diff
@@ -146,51 +147,68 @@ def execute_backfill_jobs(
 
         # refetch, in case the backfill was updated in the meantime
         backfill = cast(PartitionBackfill, instance.get_backfill(backfill_id))
-        # create a logger that will always include the backfill_id as an `extra`
-        backfill_logger = cast(
-            logging.Logger,
-            logging.LoggerAdapter(logger, extra={"backfill_id": backfill.backfill_id}),
-        )
-        try:
-            evaluation_time = pendulum.now("UTC")
-            tick = instance.create_tick(
-                TickData(
-                    instigator_origin_id=_BACKFILL_ORIGIN_ID,
-                    instigator_name=backfill_id,
-                    instigator_type=InstigatorType.BACKFILL,
-                    status=TickStatus.STARTED,
-                    timestamp=evaluation_time.timestamp(),
-                    selector_id=backfill_id,
-                )
+        evaluation_time = pendulum.now("UTC")
+
+        # TODO - probably need to scope by code location. for now just put everything together for testing
+        log_key = ["backfill", backfill_id, evaluation_time.strftime("%Y%m%d_%H%M%S")]
+        with InstigationLogger(
```

Comment:
2. sensor logs are stored by the user, for OSS we can use the same storage, but for plus we will likely need to store the logs ourselves. my homework for today is to understand how we would handle this in plus, but any ideas/opinions on how to manage this from those with more platform experience would be super helpful

Reply:
The use of the compute log manager for sensor logs is kind of a hack / implementation detail.

Compute logs are user-configured, and for a lot of our customers are not viable because they want a hard separation between Dagster and their computation layer.

We're almost better off with a new instance API call like storeLogMessage or something like that.  In Cloud,  that thing can be backed by something like some S3 storage (that is different than the user-configured compute log manager) so that these messages never touch Postgres.

In OSS, we can use the compute log manager or another configured instance class to put these logs someplace durable.